docs/ossm/create-mesh/README.md

```diff
@@ -0,0 +1,99 @@
+
+### DiscoverySelectors
+Discovery selectors provide a mechanism for the mesh administrator to limit the scope of a service mesh. This is done through a Kubernetes [label selector](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors), which defines criteria for which namespaces will be visible to the service mesh control plane. Any namespaces not matching are ignored by the control plane entirely.
+
+> **_NOTE:_** Istiod will always open a watch to OpenShift for all namespaces. However, discovery selectors will ignore objects that are not selected very early in its processing, minimizing costs.
+
+> **_NOTE:_** `discoverySelectors` is not a security boundary. Istiod will continue to have access to all namespaces even when you have configured your `discoverySelectors`.
+
+ #### Using DiscoverySelectors
+The `discoverySelectors` field accepts an array of Kubernetes [selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements). The exact type is `[]LabelSelector`, as defined [here](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements), allowing both simple selectors and set-based selectors. These selectors apply to labels on namespaces.
+
+You can configure each label selector for a variety of use cases, including but not limited to:
+
+- Arbitrary label names/values, for example, all namespaces with label `istio-discovery=enabled`
+- A list of namespace labels using set-based selectors which carries OR semantics, for example, all namespaces with label `istio-discovery=enabled` OR `region=us-east1`
+- Inclusion and/or exclusion of namespaces, for example, all namespaces with label `istio-discovery=enabled` AND label key `app` equal to `helloworld`
+
+#### Using Discovery Selectors to Scope of a Service Mesh
+Assuming you know which namespaces to include as part of the service mesh, as a mesh administrator, you can configure `discoverySelectors` at installation time or post-installation by adding your desired discovery selectors to Istio’s MeshConfig resource. For example, you can configure Istio to discover only the namespaces that have the label `istio-discovery=enabled`.
+
+##### Prerequisites
+- The OpenShift Service Mesh operator has been installed
+- An Istio CNI resource has been created
+
+1. Create the `istio-system` system namespace and create the Istio CR with `discoverySelectors` configured:
```

Comment:
As discussed by Gwynne, we'll need to break this up into individual commands/steps with a description for each.

Reply:
Done